### PR TITLE
docs(project): adding project specific files

### PR DESCRIPTION
**Pull request by mynktl**
Changes:
- This PR adds project specific files to guide contributors
    List of the files is as below: 
     - [.github/ISSUE_TEMPLATE/bug_report.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/.github/ISSUE_TEMPLATE/bug_report.md)
     - [.github/ISSUE_TEMPLATE/feature-request.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/.github/ISSUE_TEMPLATE/feature-request.md)
     - [.github/pull_request_template.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/.github/pull_request_template.md)
     - [CODE_OF_CONDUCT.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/CODE_OF_CONDUCT.md)
     - [CONTRIBUTING.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/CONTRIBUTING.md)
     - [GOVERNANCE.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/GOVERNANCE.md)
     - [RELEASE.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/RELEASE.md)
     - [code-standard.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/code-standard.md)
     - [developer-setup.md](https://github.com/openebs/cstor/blob/f6c60891c02d7e38bd21343f151ccfbbd2d9e0df/developer-setup.md)

Following files were removed due to duplication:
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/ISSUE_TEMPLATE.md

Signed-off-by: mayank <mayank.patel@mayadata.io>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands and cstor pod logs will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.)
+
+* `kubectl logs deployment/maya-apiserver -n openebs`
+* `kubectl logs deployment/<pv-name>-target -n <openebs_namespace> -c cstor-istgt`
```
Comment (vishnuitta) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: does this works with 'deployment'?

**Review by vishnuitta — DISMISSED** [review]
changes are good

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mittachaitu) [reply .github/ISSUE_TEMPLATE/bug_report.md:19]: It will work

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* `kubectl get pods -n openebs`
```
Comment (mittachaitu) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: IMO good to get the output of pool configurations
`kubectl get csp -l openebs.io/storage-pool-claim=<spc_name> -o yaml`

File: README.markdown [hunk 3]
```diff
@@ -2,16 +2,17 @@
 [![codecov](https://codecov.io/gh/zfsonlinux/zfs/branch/master/graph/badge.svg)](https://codecov.io/gh/zfsonlinux/zfs)
```
Comment (mittachaitu) [inline README.markdown:2]: Can we rename this file to README.md as we are following in all other places?

Comment (mynktl) [reply README.markdown:2]: we can rename it, but i don't think it is required. Extension of the file can be different. If you want we can rename it in a different PR.

**Review by mittachaitu — DISMISSED** [review]
Changes are good

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* `kubectl get cvr,spc,csp,cstorvolume -A -o yaml`
```
Comment (mittachaitu) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: For reporting issue, I think we need logs of cstor-pool container also

Comment (mynktl) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: yes, i've mentioned in above comment.

Comment (mittachaitu) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: that is target pod istgt container right?

Comment (mynktl) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: https://github.com/openebs/cstor/pull/301/files/a7d4c5c80c518e44fbb2cfcf890a5bfafa8dfb40#diff-74c2649c999338ed7a2833d79d4b25c3R15

Comment (mittachaitu) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: Ok!! I am searching for command...

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Release Process
+cstor follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/openebs/openebs/projects).
```
Comment (kmova) [inline RELEASE.md]: Link should be: https://github.com/orgs/openebs/projects

Comment (mynktl) [reply RELEASE.md]: done

Comment (mynktl) [reply RELEASE.md]: for other repo i've drafted PR:
https://github.com/openebs/libcstor/pull/53
https://github.com/openebs/velero-plugin/pull/71
We will push further doc related fixes to above pr and will proceed with merge during RC1.

File: RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Once the release branch is created, changelog from `changelogs/unreleased` needs to be moved to release specific folder `changelogs/v1.9.x`, if release branch is `v1.10.x` then folder will be `changelogs/v1.10.x`.
+
+The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: v1.9.0-RC1 is a GitHub release tag for cstor release build. v1.9.0 is the release tag that is created after the release criteria are satisfied by the cstor builds.)
+
+Once the release is triggered, Travis build process has to be monitored. Once Travis build is passed images are pushed to docker hub and quay.io. Images can be verified by going through docker hub and quay.io. Also the images shouldn't have any high-level vulnerabilities.
+
+Images are published at the following location:
+https://quay.io/repository/openebs/cstor?tab=tags
+https://hub.docker.com/r/openebs/cstor/tags
+
```
Comment (mittachaitu) [inline RELEASE.md]: We need to add arm builds also `https://hub.docker.com/r/openebs/cstor-pool-arm64/tags`(I missed in istgt I will add it)

Comment (mynktl) [reply RELEASE.md]: done

**Review by kmova — APPROVED** [review]

**Pull request merged**